src/main.spec.js

```diff
@@ -90,5 +90,61 @@ test("Checks HTMLElements", () => {
       );
       expect(child.classList.contains("fs-api-entry")).toBe(true);
     }
+    if (tree.children) {
+      if (tree.previousSibling) {
+        //At first we expect from the li not to contain the "collapse" class, since its contents are diplayed.
+        expect(
+          tree.parentNode.classList.contains("fs-api-directory-collapse")
+        ).toBe(false);
+        //We click the folder.
+        tree.previousSibling.click();
+        //Now we expect that the list collapses
+        expect(
+          tree.parentNode.classList.contains("fs-api-directory-collapse")
+        ).toBe(true);
+        //We click the folder.
+        tree.previousSibling.click();
+        //Now we expect for the folders' contents to be displayed.
+        expect(
+          tree.parentNode.classList.contains("fs-api-directory-collapse")
+        ).toBe(false);
+      }
+    }
   }
 });
+
+describe("Tests fetch call", () => {
```

Comment:
We have to rename this to something more semantic. Usually we prefer adding the method/function name we test in the `describe` call. For example:

```javascript
describe("#renderUrl", () => // ...
```